api/service.go

```diff
@@ -640,10 +803,20 @@ func (s *service) AddFile(server pb.API_AddFileServer) error {
 		return err
 	}
 
-	log.Debugf("stored file with path: %s", pth.String())
+	log.Debugf("added file %s to folder with new path: %s", pth.String(), dirpth.String())
 	return nil
 }
 
+func parsePath(pth string) (folder, name string) {
+	pth = strings.TrimPrefix(pth, "/")
+	parts := strings.Split(pth, "/")
+	folder = parts[0]
+	if len(parts) > 1 {
+		name = filepath.Join(parts[1:]...)
+	}
```

Comment:
Maybe `strings.SplitN(..., 2)` can be used to avoid the `if`.

Reply:
I've found the go `path` package useful for these types of things.

Reply:
Doesn't `strings.SplitN(..., 2)` return _at most 2 substrings_? As in, it still could be just one?

Reply:
I made my comment quite sloppy, I meant the thing inside the if.
Yep, will return at most two substrings. 
The second one would be the `filepath.Join(...)` for free. (`Split` separated things that it later rejoins again).

Reply:
Oh yeah! Of course 🤦‍♂